assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/TotalUserCount.js

```diff
@@ -115,7 +115,7 @@ export default function TotalUserCount( { dimensionName, dimensionValue } ) {
 						height={ 17 }
 					/>
 
-					{ numberFormat( Math.abs( change ), { style: 'percent', maximumFractionDigits: 1 } ) }
+					{ numFmt( Math.abs( change ), { style: 'percent', maximumFractionDigits: 2 } ) }
```

Comment:
The `maximumFractionDigits: 1` was actually correct here. All our "change percentages" use a single digit for the maximum. Only values where the "primary value" is a percentage use 2 digits. I'll update that before merging.